### PR TITLE
Refactor/fix travis building

### PR DESCRIPTION
**Pull request by mrsasha**
@falkorichter @skraynick chk pls!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ before_script:
   - android-wait-for-emulator
   - adb shell input keyevent 82 &
 
-script: ./gradlew assemble connectedAndroidTest
```
Comment (skraynick) [inline .travis.yml]: Why not just delete it rather than comment out. If we need it later we can add it back.

File: .travis.yml [hunk 2]
```diff
@@ -26,4 +26,4 @@ before_script:
...
+# script: ./gradlew assemble connectedAndroidTest
```
Comment (skraynick) [inline .travis.yml]: Why not just delete it rather than comment out. If we need it later we can add it back.
 Add a line note

Comment (mrsasha) [inline .travis.yml]: I removed it now, because I'm adding it in another branch (failing test fix)

**Comment by skraynick** [comment]
@mrsasha great. I will merge.

**Pull request merged**